src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import core.basesyntax.service.IFileReader;
+import core.basesyntax.service.IFileWriter;
+import core.basesyntax.service.impl.EventHandler;
+import core.basesyntax.service.impl.FileReaderCvs;
+import core.basesyntax.service.impl.FileWriterCvs;
+
+public class Main {
+    public static void main(String[] args) {
+        IFileReader cvsReader = new FileReaderCvs();
+        IFileWriter cvsWriter = new FileWriterCvs();
+        EventHandler eventHandler = new EventHandler();
+
+        var list = cvsReader.read("src/main/java/core/basesyntax/resources/activities.csv");
+        eventHandler.convertList(list);
+        cvsWriter.write("src/main/java/core/basesyntax/resources/report.csv");
```

Comment:
- do not use **var**
- use constants for file names
- 
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/f0757f34-d36e-41d8-8697-85ea4dc3a5ef)

- see main() method structure and implement it
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/3cc3708c-6fca-4a83-9a10-7801b63dd2c4)

Reply:
Sorry for the first commit, I didn't see the additional explanation in the README file